package.json

```diff
@@ -6,7 +6,7 @@
     "mockgen": "./dist/index.js"
   },
   "scripts": {
-    "build": "npm run clean && tsc",
+    "build": "tsc",
```

Comment:
Remove the `dist` directory before building. We don't want any previously files to be lying around there.